### PR TITLE
Add Debugging section to CCIE docs

### PR DESCRIPTION
**Pull request by zzak**
Includes tips for Replicated provided by @kimh

### CODE REVIEW

**Comment by kimh** [comment]
I would say how to restart replicated apps is worth for being exposed in the doc but other things are a bit too ad-hoc to be in our public doc. I had to try these things in the previous debugging session because support bundle was missing many files but it normally should contain all these information (i.g. replicated version, the output of docker ps, etc).

**Comment by felicianotech** [comment]
I made a small comment. So far, syntax wise the doc seems good. I wouldn't know too much on the topic itself.

**Comment by zzak** [comment]
@kimh @felicianotech Updated, thanks for the :eyes:

**Comment by kimh** [comment]
LGTM

File: jekyll/_docs/enterprise/debugging.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+```
+sudo restart replicated-ui
+sudo restart replicated
+sudo restart replicated-agent
+```
+
+Then, go to your services box admin (i.e.: https://YOUR-CCIE-INSTALL:8800) and try restarting with "Stop Now" and "Start Now".
+
+### Trying to login replicated
```
Comment (ndintenfass) [inline jekyll/_docs/enterprise/debugging.md]: We should capitalize Replicated - here and in the line below

**Comment by ndintenfass** [comment]
Given that this is all about Replicated I'd suggest we make the title Debugging Replicated or just Replicated -- as a general Debugging page this will disappoint, but as a way to get info about Replicated it seems reasonable (please capitalize Replicated, per my inline comments).

**Comment by zzak** [comment]
@ndintenfass Thanks for your feedback!

**Comment by bellkev** [comment]
Discussion of where logs are?

**Pull request merged**